### DISCUSSION TITLE
upgrade from 2.4.50 to 2.4.60 /boot disk space issue

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#12933] [2.4] [answered]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

24

### RAM

128

### Storage for /

500GB

### Storage for /nsm

6TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi All,

upon upgrading from 2.4.50 to 2.4.60, the upgrade process fails due to lack of space in the /boot partition. it says it needs 7mb.
i ls this directory and i found files with versions and with early dates which i think are there because no cleaning is done after setup.
i have moved files that i think i do not need and it helped solving the problem eventually.

Regards,
Udi

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
If your system was originally installed with an ISO image from before 2.4.40, the the /boot partition should be 512MB. Starting in our 2.4.40 ISO image, the /boot partition is set to 1GB to help avoid this issue.
https://docs.securityonion.net/en/2.4/release-notes.html#id43
https://github.com/Security-Onion-Solutions/securityonion/issues/12002

You should be able to remove old kernel packages to free up space if necessary.
